tests/ui/delegation/not-supported.rs

```diff
@@ -70,12 +70,16 @@ mod opaque {
 
         pub fn opaque_arg(_: impl Trait) -> i32 { 0 }
         pub fn opaque_ret() -> impl Trait { unimplemented!() }
+        //~^ warn: this function depends on never type fallback being `()`
+        //~| warn: this was previously accepted by the compiler but is being phased out; it will become a hard error in a future release!
```

Comment:
Is there a way to show what error would be emitted?

Reply:
I've implemented a rudimentary attempt at https://github.com/rust-lang/rust/pull/126367